README.md

```diff
@@ -96,3 +96,13 @@ see [.github/workflows/k8s-manifests-debug-test.yaml](.github/workflows/k8s-mani
   with:
     kubeconfig: ${{ secrets.SOCIALGOUV_KUBE_CONFIG_DEV }}
 ```
+
+## `socialgouv/actions/k8s-deactivate`
+
+- Clean review branches whenever a pull request is closed.
```

Comment:
```suggestion
- Destroy kube namespace whenever a pull request is closed and a branch destroyed
```

Reply:
`and the branch destroyed` ... that's not true. Deactivation is triggered on PR close event only.